content/themis/languages/python/installation.md

```diff
@@ -75,13 +75,151 @@ Once PyThemis is installed, you can [try out code examples](../examples/).
+
+    Depending on how you want to install PyThemis, there are different requirements.
+
+    To create a `.whl` package for installation inside virtualenv, you'll need `setuptools` and `wheel`
+
+    * Install globally
+
+      For Debian, Ubuntu:
+
+      ```bash
+      sudo apt install python3-setuptools python3-wheel
+      ```
+
+<!--
+      For CentOS, RHEL:
+
+      ```bash
+      sudo yum install python3-setuptools python3-wheel
+      ```
+-->
+
+    * Or install into virtualenv
+
+      ```bash
+      # activate virtualenv if not done yet
+      pip install setuptools wheel
+      ```
+
+    ---
+
+    To create a system package (i.e. `.deb` one for Debian) you will need `pip` and `fpm`
+
+    * Install globally
+
+      For Debian, Ubuntu:
+
+      ```bash
+      sudo apt install python3-pip lsb-release ruby
+      sudo gem install fpm
+      ```
+
+<!--
+      For CentOS, RHEL:
+
+      ```bash
+      sudo yum install python3-pip ruby
+      sudo gem install fpm
+      ```
+
+    On RHEL you also need to have working `pip` command. If it's not, and you only got `pip3`,
+    create a symlink like this: `sudo ln -s $(which pip3) /usr/bin/pip`.
+-->
+
+ 3. Create a PyThemis package
+
+    * Create a "wheel" for virtualenv
+
+      {{< hint info >}}
+      If you installed dependencies globally, virtualenv should not be activated during wheel
+      generation since needed packages won't be visible in an isolated environment.
+      {{< /hint >}}
+
+      ```bash
+      make pythemis_make_wheel
+      ```
+
+      Result package filename will be printed at the end, like this:
+      ```
+      Result: src/wrappers/themis/python/dist/pythemis-0.14.0-py2.py3-none-any.whl
+      ```
+
+    * Create package for your distro
+
+      For Debian, Ubuntu:
+
+      ```bash
+      make pythemis_deb
+      ```
+
+<!--
+      For CentOS, RHEL:
+
+      ```bash
+      make pythemis_rpm
+      ```
+
+      The result will be located at `build/rpm/python3-pythemis_..._all.rpm`
+      or `build/rpm/python3-pythemis_..._all.rpm` respectively.
+-->
+
+ 4. Install PyThemis
+
+    * Install a wheel into virtualenv
+
+      ```bash
+      # activate virtualenv if not done yet
+      make pythemis_install_wheel
+      ```
+
+      Or manually run `pip install path/to/pythemis-...-none-any.whl` (the package file previously created).
+      If the virtual environment already contained PyThemis of the same version,
+      add `--force-reinstall` option to rewrite the previous package.
+
+    * Install system package
+
+      For Debian, Ubuntu:
+
+      ```bash
+      make pythemis_install_deb
+      ```
+
+      For manual installation, there are two ways:
+      1. `sudo apt install ./path/to/package.deb` (the `./` is important here!)
+      2. `sudo dpkg -i path/to/package.deb`
+
+<!--
+      For CentOS, RHEL:
+
+      ```bash
+      make pythemis_install_rpm
+      ```
+-->
+
+## Building latest version from source (deprecated)
```

Comment:
I don't like this title for this block. As I understand, we need it for users who want install old version of the themis from the sources. And when they will pull old version, they will not have new makefile that knows how to pack .whl/.deb/.rpm.

@vixentael , what do you think, do we need instructions how to install old versions themis from the sources? If yes, how you suggest this title? Below we will describe how install pythemis from the sources themis < 0.16

Reply:
I'd leave this instruction for older Themis, but with note that it's for old Themis.

```suggestion
## Building Themis < 0.15 version from sources (deprecated)

{{< hint warning >}}
The instructions below are relevant only for Themis older than 0.15. Most likely, you don't need to follow them, and should install the latest Themis instead.
{{< /hint >}}
```